### DISCUSSION TITLE
How to generate PGN with variations?

### DISCUSSION THREAD

**Discussion by govind-maheshwari2** [alexobviously/bishop#79] [Q&A]
Req. PGN
```
[FEN "r1b2r1k/pp1p2pp/5n1N/8/P7/1Q6/1PP3PP/4R2K b - - 0 1"]
[SetUp "1"]
 1... Rb8 2. Re8 Rxe8 (2... g6 3. Qf7 Ng8 4. Qxg8+ Rxg8 5. Rxg8# $300) 3. Qg8+ Nxg8 4. Nf7#
```

**Comment by alexobviously** [comment]
We don't have this feature yet, it's on the roadmap but not very far up. However, if this is something you really need, I can prioritise it.

**Reply by govind-maheshwari2** [reply]
Yes I need this feature a little bit more urgent. If you can priorities this it would be really helpful.

**Reply by alexobviously** [reply]
Alright, then I'll put it on my list and probably get to it next week.

**Reply by govind-maheshwari2** [reply]
Hi @alexobviously,
Any update on this?
